src/pages/UsersPage.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components/macro'
+import User from '../components/User'
+
+const data = require('../data/data.json')
```

Comment:
I think it is better to just import the data once and then pass it into the nested component. (in your `User`-component you also import the data)

Reply:
Yup, import it here (or in the `App.js`), get rid of it in _all_ other files.